examples/basic_phoenix_ecto/docker-compose.yaml

```
@@ -0,0 +1,40 @@
+version: '3'
+services:
+  postgres:
+    image: 'postgres:latest'
+    ports:
+      - 5432:5432
+
+    environment:
+      POSTGRES_USER: postgres
+      POSTGRES_PASSWORD: postgres
+      POSTGRES_DB: demo_dev
+
+  phoenix:
+    env_file: .env
+    build: .
+    ports:
+      - 4000:4000
+
+
+  otel:
+    image: otel/opentelemetry-collector-contrib-dev:latest
+    command: ["--config=/etc/otel-collector-config.yaml"]
+    ports:
+      - '55681:55681'
+      - '55680:55680'
```

I think those ports will be deprecated based on collector logs

```
otel-collector_1  | 2022-02-03T20:31:46.586Z	info	builder/receivers_builder.go:68	Receiver is starting...	{"kind": "receiver", "name": "otlp"}
otel-collector_1  | 2022-02-03T20:31:46.586Z	info	otlpreceiver/otlp.go:87	Starting HTTP server on endpoint 0.0.0.0:4318	{"kind": "receiver", "name": "otlp"}
otel-collector_1  | 2022-02-03T20:31:46.586Z	info	otlpreceiver/otlp.go:147	Setting up a second HTTP listener on legacy endpoint 0.0.0.0:55681	{"kind": "receiver", "name": "otlp"}
otel-collector_1  | 2022-02-03T20:31:46.586Z	info	otlpreceiver/otlp.go:87	Starting HTTP server on endpoint 0.0.0.0:55681	{"kind": "receiver", "name": "otlp"}
otel-collector_1  | 2022-02-03T20:31:46.586Z	info	builder/receivers_builder.go:73	Receiver started.	{"kind": "receiver", "name": "otlp"}
```

```suggestion
      - '4318:4318'
```
Also need to change `config/runtime.exs`

---

https://github.com/open-telemetry/opentelemetry-collector/tree/main/receiver/otlpreceiver#writing-with-httpjson

---

Thanks @jhonndabi 💯

I have make the changes accordingly.